worlds/civ_6/Regions.py

```diff
@@ -0,0 +1,145 @@
+        "ERA": world.options.progression_style
+        == world.options.progression_style.option_eras_and_districts,
+        "GOODY": world.options.shuffle_goody_hut_rewards.value,
+        "BOOST": world.options.boostsanity.value,
+    }
+
+    regions: List[Region] = []
+    for era in EraType:
+        era_region = Region(era.value, world.player, world.multiworld)
+        era_locations: Dict[str, Optional[int]] = {}
+
+        for key, location in world.location_by_era[era.value].items():
+            category = key.split("_")[0]
+            if optional_location_inclusions.get(category, True):
+                era_locations[location.name] = location.code
+
+        era_region.add_locations(era_locations, CivVILocation)
+
+        regions.append(era_region)
+        world.multiworld.regions.append(era_region)
+
+    menu.connect(world.get_region(EraType.ERA_ANCIENT.value))
+
+    world.get_region(EraType.ERA_ANCIENT.value).connect(
+        world.get_region(EraType.ERA_CLASSICAL.value),
+        None,
+        lambda state: has_required_items(state, EraType.ERA_ANCIENT, world),
+    )
+
+    world.get_region(EraType.ERA_CLASSICAL.value).connect(
+        world.get_region(EraType.ERA_MEDIEVAL.value),
+        None,
+        lambda state: has_required_items(state, EraType.ERA_CLASSICAL, world),
+    )
+
+    world.get_region(EraType.ERA_MEDIEVAL.value).connect(
+        world.get_region(EraType.ERA_RENAISSANCE.value),
+        None,
+        lambda state: has_required_items(state, EraType.ERA_MEDIEVAL, world),
+    )
+
+    world.get_region(EraType.ERA_RENAISSANCE.value).connect(
+        world.get_region(EraType.ERA_INDUSTRIAL.value),
+        None,
+        lambda state: has_required_items(state, EraType.ERA_RENAISSANCE, world),
+    )
+
+    world.get_region(EraType.ERA_INDUSTRIAL.value).connect(
+        world.get_region(EraType.ERA_MODERN.value),
+        None,
+        lambda state: has_required_items(state, EraType.ERA_INDUSTRIAL, world),
+    )
+
+    world.get_region(EraType.ERA_MODERN.value).connect(
+        world.get_region(EraType.ERA_ATOMIC.value),
+        None,
+        lambda state: has_required_items(state, EraType.ERA_MODERN, world),
+    )
+
+    world.get_region(EraType.ERA_ATOMIC.value).connect(
+        world.get_region(EraType.ERA_INFORMATION.value),
+        None,
+        lambda state: has_required_items(state, EraType.ERA_ATOMIC, world),
+    )
+
+    future_era = world.get_region(EraType.ERA_INFORMATION.value)
+    future_era.connect(
+        world.get_region(EraType.ERA_FUTURE.value),
+        None,
+        lambda state: has_required_items(state, EraType.ERA_INFORMATION, world),
+    )
+
+    victory = CivVILocation(world.player, "Complete a victory type", None, future_era)
+    victory.place_locked_item(world.create_event("Victory"))
+    future_era.locations.append(victory)
+
+    set_rule(
+        victory,
+        lambda state: state.can_reach(EraType.ERA_FUTURE.value, "Region", world.player),

```

Comment:
I didn't catch this previously, but there's actually a function called `state.can_reach_region(region_name, player)` that is slightly preferred.

*low priority*